app/bibleview-js/src/lang/default.yaml

```diff
@@ -1,5 +1,5 @@
 openMyNotes: Open in My Notes
-openStudyPad: Open in Study Pad (%s)
+openStudyPad: "Open '%s' study pad"
```

Comment:
These changes I think we could do in Transifex (en_GB)

Reply:
Done